lib/spree_i18n.rb

```diff
@@ -1,4 +1,6 @@
 require 'rails-i18n'
+require 'rails/generators'
```

Comment:
Are you sure we need this line? I think only the second one is required, based on this comment: https://github.com/rails/rails/issues/12802#issuecomment-28095045 Generator for spree_i18n also uses only Rails::Generators::Base constant (https://github.com/spree-contrib/spree_i18n/blob/master/lib/generators/spree_i18n/install/install_generator.rb#L3)